crates/hc_sandbox/README.md

```diff
@@ -1,110 +1,249 @@
+hc sandbox generate --root my-sandboxes/
+# three explicitly named sandboxes in the system temp directory
+hc sandbox generate --num-sandboxes 3 --directories alice,bob,carol
+```
+
+While these directories are useful for creating test fixtures, their usefulness on other machines and across test runs is limited, as the generated conductor configurations have machine-dependent paths and the databases accumulate data with each run.
+
+Finally, you can specify the type of network transport the sandboxes will use to communicate with each other via a `network` subcommand at the very end.
+
+```shell
+hc sandbox generate network quic
+```
+
+You can also generate and run in the same command using the `--run` option. The argument passed to `-r` is a comma-separated list of ports to bind the sandboxes' app API WebSockets to, with `0` indicating that a port should be auto-selected. Once again, make sure the number of ports matches the number of sandboxes to be run; if not enough ports are specified, the remaining sandboxes won't be run.
+
 ```shell
- hc sandbox gen -a "my-app" -n 5 ./elemental-chat.dna network quic
+hc sandbox generate --num-sandboxes 5 --run 0,9500,9501,0,0 ./elemental-chat.happ
 ```
-You can also generate and run in the same command:
-(Notice the number of conductors and dna path must come before the gen sub-command).
+
+As a full example, this will generate and run five named sandboxes in a subdirectory called `my-sandboxes`, with app IDs set to `my-app`
+using the `elemental-chat.happ` from the current directory with a QUIC
+network configured to use `localhost`.
+
+_You don't need to specify the filename of the hApp when there's only one in your current working directory._
+
 ```shell
- hc sandbox r -n 5 ./elemental-chat.dna gen -a "my-app" network quic
+hc sandbox generate \
+    --app-id "my-app" \
+    --num-sandboxes 5 \
+    --root my-sandboxes/ \
+    --directories alice,bob,carol,dave,eve \
+    --run 0,0,0,0,0 \
+    ./elemental-chat.happ \
+    network quic
+```
+
+#### Create
+
+Creates 'empty' sandboxes; that is, sandboxes with no apps installed. This can be useful for testing the implementation of a program that controls the conductor via the admin API, such as an application launcher. Most of the options for `hc generate` also work with `hc create`:
+
+```shell
+hc sandbox create \
+    --num-sandboxes 5 \
+    --root my-sandboxes/ \
+    --directories alice,bob,carol,dave,eve \
+    --run 0,0,0,0,0 \
+    network quic
+```
+
+#### Call
+
+Allows calling the [conductor admin API](https://docs.rs/holochain_conductor_api/latest/holochain_conductor_api/enum.AdminRequest.html) API on one or more _running_ sandboxes.
+Although the API functions receive input as MessagePack-serialized data,
+this command lets you conveniently pass them as command-line arguments instead.
+For a list of all available admin API functions, run:
+
+```shell
+hc sandbox call --help
+```
+
+```text
+[... options and flags ...]
```

Comment:
I recommend to not replicate detailed output from help commands here and refer to `--help` instead.